content/TrueCommand/Administration/Settings/SAML.md

```diff
@@ -42,149 +40,148 @@ To configure AD as the IdP, you must create and modify a Relying Party Trust. Af
 
          ![OpenTcCertificate](/images/SAML/OpenTcCertificate.png "OpenTcCertificate")
         
-      4. Type the following string exactly as `-----BEGIN CERTIFICATE-----` before the certificate with five dashes before and after..
-      5. Type the following string exactly as `-----END CERTIFICATE-----` after the certificate with five dashes before and after.
+      iv. Type the following string exactly as `-----BEGIN CERTIFICATE-----` before the certificate with five dashes before and after.
+      v. Type the following string exactly as `-----END CERTIFICATE-----` after the certificate with five dashes before and after.
       
          ![EditedTcCertificate](/images/SAML/EditedTcCertificate.png "Edited Tc Certificate")
       
-      6. Select Save (or Ctrl-S) and then close Notepad.
+      vi. Click **Save** (or <kbd>Ctrl-S</kbd>) and then close **Notepad**.
 
    c. Close Powershell.
 
    d. Select Browse on the Configure Certificate window. The Encryption Certificate window opens. 
 
       ![WizardConfigureCertificateSelectBrowseCropped](/images/SAML/WizardConfigureCertificateSelectBrowseCropped.png "Wizard Configure Certificate Select Browse Cropped")
     
-   e. Locate the tc.cer file (c:/local disk/users/QE and then select the tc file. Click Open. The Configure Certificate window displays information about the certificate. Click **Next** and the Configure URL window displays.
+   e. Locate the <file>tc.cer</file> file (**C:/local disk/users/QE** and then select the <file>tc.cer</file>. Click **Open** to view the **Configure Certificate** window and see information about the certificate. Click **Next** to move on to the **Configure URL** window.
 
-4. Configure the URL. On the Configure URL window:
+4. Configure the URL. In the **Configure URL** window:
 
-   a. Select Enable support for the SAML 2.0 webSSO protocol. 
+   a. Select **Enable support for the SAML 2.0 WebSSO protocol**. 
         
    ![WizardSelectEnableSupportForSAML2_0WebSSOProtocolCropped](/images/SAML/WizardSelectEnableSupportForSAML2_0WebSSOProtocolCropped.png "Wizard Select Enable Support For SAML 2.0 Web SSO Protocol Cropped")
         
-   b. Type or copy/paste the TrueCommand login URL http://IP:PORT/saml/acs in the Relying party SAML 2.0 SSO service URL field. Where IP:PORT is your TrueCommand system IP and port address.
+   b. Type or copy/paste the TrueCommand login URL (*http://IP:PORT/saml/acs*) in the **Relying party SAML 2.0 SSO service URL** field. *IP:PORT* is your TrueCommand system IP and port address.
 
-   c. Click **Next**. The Configure Identifiers window displays.
+   c. Click **Next** to continue to the **Configure Identifiers** window.
 
 5. Configure the SAML identifiers.
 
-   a. Type truecommand-sml into the Relying party trust identifier field and click Add.
+   a. Type *truecommand-sml* into the **Relying party trust identifier** field and click **Add**.
```

Comment:
truecommand-sml is not a variable in this procedure, suggest adding bold as it is the name of the SAML service in TrueCommand.
   a. Type **truecommand-sml** into the **Relying party trust identifier** field and click **Add**.

Reply:
Done